src/aspire/source/__init__.py

```diff
@@ -526,24 +529,174 @@ def vol_forward(self, vol, start, num):
         im *= self.amplitudes[all_idx, np.newaxis, np.newaxis]
         return im
 
-    def save(self, starfile_filepath, batch_size=512, save_mode=None, overwrite=False):
+    def save(
+        self,
+        starfile_filepath,
+        batch_size=512,
+        save_images=True,
```

Comment:
No need for a flag here. If a user only wants to save images or only wants to save metadata, they can use `save_images` and `save_metadata` directly.

Reply:
Done in commit c1dae2b.

Reply:
The flag is still there.

Reply:
Removed from the `save` function but added to `save_metadata` as the flag suggested below.